src/index.html

```diff
@@ -20,8 +20,305 @@
+          <li class="header__nav-item">
+            <a
+              href="#gadgets"
+              class="header__nav-link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#tablets"
+              class="header__nav-link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#photo"
+              class="header__nav-link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#video"
+              class="header__nav-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </div>
+    </header>
+    <main class="main">
+      <div
+        data-qa="card"
+        class="card-grid"
+      >
+        <div class="card">
+          <div class="card__img"></div>
+          <a
+            href="#Apple"
+            class="card__name"
+          >
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </a>
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="stars">
+            <div class="stars__icon stars__icon--active"></div>
+            <div class="stars__icon stars__icon--active"></div>
+            <div class="stars__icon stars__icon--active"></div>
+            <div class="stars__icon stars__icon--active"></div>
+            <div class="stars__icon"></div>
+            <div class="stars__reviews">Reviews: 5</div>
+          </div>
+
+          <div class="price">
+            <div class="price-text">Price:</div>
+            <div class="price__value">$2,199</div>
+          </div>
+
+          <button
+            data-qa="card-hover"
+            class="card__button"
+          >
+            BUY
+          </button>
+        </div>
+        <div class="card">
+          <div class="card__img"></div>
+          <a
+            href="#"
+            class="card__name"
+          >
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </a>
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="stars">
+            <div class="stars__icon stars__icon--active"></div>
+            <div class="stars__icon stars__icon--active"></div>
+            <div class="stars__icon stars__icon--active"></div>
+            <div class="stars__icon stars__icon--active"></div>
+            <div class="stars__icon"></div>
+            <div class="stars__reviews">Reviews: 5</div>
+          </div>
+
+          <div class="price">
+            <div class="price-text">Price:</div>
+            <div class="price__value">$2,199</div>
+          </div>
+
+          <button class="card__button">BUY</button>
+        </div>
+        <div class="card">
+          <div class="card__img"></div>
+          <a
+            href="#"
```

Comment:
The 'href' attribute is set to '#', which is a placeholder. Consider replacing it with an actual URL to ensure proper navigation.